### ISSUE TITLE
Incompatibility with Rom 3.2.0

### ISSUE THREAD

**Issue by mrship** [rom-rb/rom-yesql#13]
Having upgraded Rom to 3.2.0, we get this error now when initialising `rom-yesql`

Other than dropping back to Rom 3.1.0; is there a workaround?

```
ArgumentError: wrong number of arguments (0 for 1)
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-yesql-0.4.0/lib/rom/yesql/gateway.rb:29:in `block in <class:Gateway>'
  (eval):6:in `instance_exec'
  (eval):6:in `block in __initialize__'
  (eval):6:in `fetch'
  (eval):6:in `__initialize__'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/dry-initializer-1.3.0/lib/dry/initializer.rb:36:in `initialize'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-yesql-0.4.0/lib/rom/yesql/gateway.rb:71:in `initialize'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/gateway.rb:78:in `new'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/gateway.rb:78:in `setup'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:51:in `block in normalize_gateways'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:45:in `each'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:45:in `each_with_object'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:45:in `normalize_gateways'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:25:in `configure_gateways'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/environment.rb:18:in `initialize'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/configuration.rb:24:in `new'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/configuration.rb:24:in `initialize'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/create_container.rb:48:in `new'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/create_container.rb:48:in `initialize'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/create_container.rb:58:in `new'
  /Users/shipmana/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/rom-3.2.0/lib/rom/create_container.rb:58:in `container'```

**Comment by flash-gordon** [comment]
I'll update it to work with 3.2 today, thank for reporting

**Comment by flash-gordon** [comment]
@mrship 0.5.0 is on rubygems 🎉

**Comment by mrship** [comment]
Much obliged. This (obviously) doesn't work with 3.1.0 though so it might be worthwhile adding a fixed dependency on rom > 3.2 for the next version, as we had some issues when upgrading getting the dependencies right. 

Thanks for the quick response.

**Comment by flash-gordon** [comment]
Ugh, right, thank for spotting this, I'll release 3.2.1 with a bumped dep
